composer/datasets/synthetic.py

```diff
@@ -10,6 +12,8 @@
 from composer.core.types import MemoryFormat
 from composer.utils.string_enum import StringEnum
 
+__all__ = ["SyntheticDataType", "SyntheticDataLabelType", "SyntheticBatchPairDataset", "SyntheticPILDataset"]
+
 
 class SyntheticDataType(StringEnum):
```

Comment:
Doc string this class. See https://github.com/mosaicml/composer/blob/dev/composer/core/logging/logger.py#L41-L44 for reference.

Reply:
Same for SyntheticDataLabelType